modules/appeals_api/config/schemas/10182.json

```diff
@@ -46,18 +46,50 @@
       "additionalProperties": false,
       "properties": {
         "homeless":            { "type": "boolean" },
-        "representativesName": { "type": "string", "maxLength": 120 },
-        "address": { "$ref": "#/definitions/nodCreateAddress" }
+        "address":             { "$ref": "#/definitions/nodCreateAddress" },
+        "phone":               { "$ref": "#/definitions/nodCreatePhone" },
+        "emailAddressText":    { "$ref": "#/definitions/nodCreateEmail" },
```

Comment:
add a `minLength: 3` and `maxLength` that will fit within the autosize box without truncation.

Reply:
I'll figure out truncation cut-off and update! What's the rationale for `minLength: 3` ?

Reply:
This was for email address. The pattern is looking for 3 total characters to pass, so that's what I based the `minLength` off of.

Reply:
Oh! Email, not address. Gotcha.